webpack.config.js

```diff
@@ -119,7 +119,7 @@ const webpackConfig = {
 			},
 			{
 				test: /\.s?css$/,
-				exclude: /storybook\/wordpress/,
+				exclude: [ /storybook\/wordpress/, /build-style\/*\/*.css/ ],
```

Comment:
I added this to not include all the individual component styles as well. Not sure how much of a difference it makes (or if I did it correctly).